### DISCUSSION TITLE
Requires 64-bit vmp samples

### DISCUSSION THREAD

**Discussion by VeroFess** [sandboxie-plus/Sandboxie#3099] [General]
I am the author of the sbiehide plug-in. 

The new version of vmp and sandboxie seems to be unable to reproduce the problem that the sandboxie is detected when the vmp memory protection is turned on. 

Is there any 64-bit sample program that can be provided?

**Comment by isaak654** [comment]
A new [SbieHideEx](https://github.com/0bbedCode/SbieHideEx) fork has just been made available, perhaps you could help each other?
